src/invoices/Voucher/VendorAddress/useVendor.js

```diff
@@ -0,0 +1,18 @@
+import { useQuery } from 'react-query';
+
+import { useOkapiKy } from '@folio/stripes/core';
+import { VENDORS_API } from '@folio/stripes-acq-components';
+
+export const useVendor = (vendorId) => {
+  const ky = useOkapiKy();
+
+  const { isLoading, data: vendor = {} } = useQuery(
+    [VENDORS_API, vendorId],
+    () => ky.get(`${VENDORS_API}/${vendorId}`).json(),
```

Comment:
imagine that 404 comes because no vendor with such id

Reply:
In this case, only the vendor name will be displayed as `-`. Or do you mean we need to notify an user about an error?

Reply:
shouldn't it be in try\catch block?

Reply:
`data: vendor = {}` should resolve catched errors